bootstrap/kubeadm/main.go

```diff
@@ -129,6 +130,9 @@ func InitFlags(fs *pflag.FlagSet) {
 	fs.IntVar(&clusterConcurrency, "cluster-concurrency", 10,
 		"Number of clusters to process simultaneously")
 
+	fs.IntVar(&clusterCacheTrackerConcurrency, "clustercache-tracker-concurrency", 10,
```

Comment:
```suggestion
	fs.IntVar(&clusterCacheTrackerConcurrency, "clustercachetracker-concurrency", 10,
```

I would probably use this for consistency. Basically the first part is something like the "controller" name

Reply:
The cluster-concurrency flag is now unused, we should drop it